run/testing/grpc_server_streaming.e2e_test.go

```diff
@@ -71,15 +72,22 @@ func TestGRPCServerStreamingService(t *testing.T) {
 	}
 
 	recvMsgs := 0
+	recvFailures := 0
```

Comment:
Let's change this to just `var recvFailures int` and do the same above while we're at it.